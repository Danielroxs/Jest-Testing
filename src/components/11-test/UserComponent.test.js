import React from "react";
import { render } from "@testing-library/react";
import UserComponent from "./UserComponent";

test("should match the snapshot", () => {
    const user = { name: "John Doe", email: "johndoe@example.com" };
    const { asFragment } = render(<UserComponent user={user} />);
    expect(asFragment()).toMatchSnapshot();
});