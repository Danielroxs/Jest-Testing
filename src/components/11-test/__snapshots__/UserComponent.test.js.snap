// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      John Doe
    </h1>
    <p>
      johndoe@example.com
    </p>
  </div>
</DocumentFragment>
`;
